test/lib/MangroveTest.sol

```diff
@@ -79,7 +79,7 @@ contract MangroveTest is Test2, HasMgvEvents {
     defaultTickscale: 1,
     gasprice: 40,
     gasbase: 50_000,
-    density: 10,
+    density: 2 ** 32,
```

Comment:
I'm surprised this doesn't affect more tests!

Reply:
Why/how was this density chosen?

Reply:
At random, bigger than what it was :)

Reply:
The tokens have fictional values, so the importance is that asserts can be made regarding density, not the exact value. Previously it was effectively 0.

Reply:
Ok :-) We can adjust later if needed